src/Microsoft.VisualStudio.ProjectSystem.VisualBasic.VS.UnitTests/ProjectSystem/VS/Properties/VisualBasicProjectDesignerPageProviderTests.cs

```diff
@@ -24,16 +25,33 @@ public void Constructor_DoesNotThrow()
             var pages = await provider.GetPagesAsync();
 #pragma warning restore RS0003 // Do not directly await a Task
 
+            Assert.Equal(pages.Count(), 3);
+            Assert.Same(pages.ElementAt(0), VisualBasicProjectDesignerPage.Application);
+            Assert.Same(pages.ElementAt(1), VisualBasicProjectDesignerPage.References);
+            Assert.Same(pages.ElementAt(2), VisualBasicProjectDesignerPage.Debug);
+        }
+
+        [Fact]
+        public async Task GetPagesAsync_WithPackCapability()
+        {
+            var provider = CreateInstance(ProjectCapability.Pack);
+
+#pragma warning disable RS0003 // Do not directly await a Task (see https://github.com/dotnet/roslyn/issues/6770)
```

Comment:
Same question and comment about why we need to explicitly disable this; it should already be disabled.